File: crates/spur-graph/tests/fixtures/rename_corpus/rust/adversarial_02/new.rs
pub fn rust_crossover_alpha_new(value: i32) -> i32 {
    let base = value + 10;
    if base > 50 {
        base - 7
    } else {
        base + 7
    }
}

pub fn rust_crossover_beta_new(value: i32) -> i32 {
    let base = value + 10;
    if base > 50 {
        base - 7
    } else {
        base + 7
    }
}

File: crates/spur-graph/tests/fixtures/rename_corpus/python/adversarial_02/new.py
def py_crossover_alpha_new(value: int) -> int:
    base = value + 10
    if base > 50:
        return base - 7
    return base + 7


def py_crossover_beta_new(value: int) -> int:
    base = value + 10
    if base > 50:
        return base - 7
    return base + 7
